src/tests/Equinor.ProCoSys.IPO.WebApi.Tests/Authorizations/AccessValidatorTests.cs

```diff
@@ -761,6 +762,34 @@ public async Task ValidateAsync_OnGetCommentsQuery_ShouldReturnFalse_WhenNoAcces
         }
         #endregion
 
+        #region GetSavedFiltersInProject
```

Comment:
Here we have some missing (or to much) security. The GetSavedFilters are secured by project. 
None of our commands (Create, Update) are secured in same way. (I see the problem is inherited from preservation).

Make a meeting with you, me, L and E and we discuss the solution. I guess we remove the security on on GetSavedFilters, and we're happy, but we discuss it in plenum